crates/miden-lib/asm/miden/account.masm

```diff
@@ -375,10 +375,7 @@ export.get_initial_storage_commitment
     # => [INIT_STORAGE_COMMITMENT]
 end
 
-#! Gets the account storage commitment of the current account.
-#!
-#! Note that this commitment may not reflect the changes made to storage. The commitment is
-#! recomputed when `account_compute_current_commitment` is called.
+#! Computes the latest account storage commitment of the current account.
```

Comment:
Should we mention somewhere in the comment (or maybe somewhere else) that we don't always recompute the commitment. I feel like we need to describe this somewhere - but not sure where's the best place.